actix-fileadapter-rbac/Cargo.toml

```diff
@@ -8,6 +8,6 @@ workspace = ".."
 [dependencies]
 actix-web = "2.0"
 actix-rt = "1.1"
-casbin = "0.9"
+casbin = "1.1"
```

Comment:
Maybe using `casbin = { version = "1.1.0", default-features = false, features = ["incremental"] }` is better